emcc.py

```diff
@@ -2200,6 +2199,21 @@ def do_minify(): # minifies the code. this is also when we do certain optimizati
               os.unlink(js_target) # we don't need the js, it can just confuse
               os.unlink(asm_target) # we don't need the asm.js, it can just confuse
             sys.exit(0) # and we are done.
+        if opt_level >= 2:
```

Comment:
Does this optimize stage do anything different than for the asm.js case? Is there any way the code can be shared?